Tahi-project/tahi
# Bug/aperta 12248 stop hiding javascript console errors in production

# Dev ticket

JIRA issue: https://jira.plos.org/jira/browse/APERTA-12248

#### What this PR does:

This PR stops suppressing javascript console errors in production and staging environments. The test environment remains unchanged.

#### Special instructions for Review or PO:

To test this do the following:

1. Add a Preprint decision card to a paper's workflow
2. Delete Preprint letter templates 
3. Open the preprint card and select Accept or Reject
4. The card should throw an error as expected

The error thrown by the card should appear in the javascript console in red as an error for production, staging and development environments.

---

#### Code Review Tasks:

**Author tasks** (delete tasks that don't apply to your PR, this list should be finished before code review):

- [x] I have set the correct component(s) in the JIRA ticket
- [x] I have set an appropriate resolution in the JIRA ticket
- [x] I have ensured that the Heroku Review App has successfully deployed and is ready for PO UAT.


**Reviewer tasks** (these should be checked or somehow noted before passing on to PO):
